modules/reindex/src/test/resources/rest-api-spec/test/reindex/90_remote.yaml

```diff
@@ -257,3 +257,56 @@
             match:
               text: test
   - match: {hits.total: 1}
+
+---
+"Reindex from remote with broken query":
```

Comment:
This one didn't trigger the error but I kept it anyway because I figured it was nice to have it around.